src/core/Config.cpp

```diff
@@ -1915,11 +1915,13 @@ OIIO_ADD_TEST(Config, EnvCheck)
     "  - !<View> {name: Raw, colorspace: raw}\n"
     "\n";
     
-    
-    std::string test("SHOW=bar");
-    putenv((char *)test.c_str());
-    std::string test2("TASK=lighting");
-    putenv((char *)test2.c_str());
+#ifdef WINDOWS
```

Comment:
There is already a method `Platform::getenv()` in `src/core/Platform.h`, so the code could be improved by adding a `Platform::setenv()` to localize the platform differences even if it's only needed by unit tests for now.

Reply:
Added `Platform::setenv()` and changed the unit test to use it.